scripts/postinstall.js

```diff
@@ -35,20 +35,53 @@ const packageName = 'timezone';
 // init cwd is the filepath of the initiating command
 const dirPath = `${process.env.INIT_CWD}/node_modules/${packageName}/`;
 
+let targetBlockValues = {};
 try {
   // eslint-disable-next-line global-require,import/no-dynamic-require
-  themesLocaleList = require(`${process.env.INIT_CWD}/src/blocks.json`).localeList;
+  targetBlockValues = require(`${process.env.INIT_CWD}/src/blocks.json`).values;
+
+  const defaultLocalizationObject = targetBlockValues.default.siteProperties.dateLocalization;
+
+  // must have a default language and timezone
+  // redefining locale list over the defaults
+  themesLocaleList = [defaultLocalizationObject.language];
+  targetTimeZones = [defaultLocalizationObject.timeZone];
+  /*
+    "sites": {
+      "arc-demo-1": {
+      "siteProperties": {
+        "dateLocalization": {
+          "language": "es",
+          "timeZone": "Europe/Madrid"
+        },
+      }
+    }
+  */
+
+  // if sites
+  // obj with key is truthy if sites property exists
+  Object.values(targetBlockValues.sites).forEach(({ siteProperties: sitePropertyObject }) => {
+    // if site has no date localization obj
+    if (sitePropertyObject.dateLocalization) {
+      themesLocaleList.push(sitePropertyObject.dateLocalization.language);
+      targetTimeZones.push(sitePropertyObject.dateLocalization.timeZone);
+    }
+  });
+
+  // dedupe languages and timezones if any added
+  themesLocaleList = [...new Set(themesLocaleList)];
+  targetTimeZones = [...new Set(targetTimeZones)];
 } catch (err) {
   // eslint-disable-next-line no-console
-  console.log(`${process.env.INIT_CWD}/src/blocks.json`, 'not found');
+  // console.log(`${process.env.INIT_CWD}/src/blocks.json`, 'not found');
 }
 
 function unlinkSyncWithErrorLogging(targetPath) {
   try {
     fs.unlinkSync(targetPath);
   } catch (err) {
     // eslint-disable-next-line no-console
-    console.log(targetPath, 'not deleted');
+    // console.log(targetPath, 'not deleted');
```

Comment:
is there anything else that would make sense to have here instead of a console.log we arent using?

Reply:
@farleyawapo : The error that a file was not deleted feels unintuitive.  Is this an error or a warning?

Reply:
Thanks for the feedback @farleyawapo and @nschubach. sounds like we'd want to see a warning of some sort. I was initially using this for debugging. But we can keep it if we want. if you're running this locally without a src/blocks.json in the directory, we'd see a lot of warnings in normal local development

I can add a better explanation for this in the code. In the fusion compiler, any warnings or logging or erroring we use would not be bubbled up sadly because this is a nested dependency as far as I understand @badmintonkid